pyensembl/__init__.py

```diff
@@ -57,12 +58,31 @@ def genome_for_reference_name(reference_name):
 
 ensembl_grch36 = ensembl54 = cached_release(54)  # last release for GRCh36/hg18
 ensembl_grch37 = ensembl75 = cached_release(75)  # last release for GRCh37/hg19
+ensembl_grch38 = cached_release(MAX_ENSEMBL_RELEASE)  # most recent for GRCh38
 
-ensembl77 = cached_release(77)
-ensembl78 = cached_release(78)
-ensembl79 = cached_release(79)
-ensembl80 = cached_release(80)
-ensembl81 = cached_release(81)
-ensembl82 = cached_release(82)
-ensembl83 = cached_release(83)
-ensembl_grch38 = ensembl83  # most recent for GRCh38
+
+__all__ = [
```

Comment:
I thought it might be slightly better package etiquette to restrict which things get imported as '*'. I'm not totally sure if it's necessary though.

https://docs.python.org/2/tutorial/modules.html#importing-from-a-package